src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/QCallHandles.cs

```diff
@@ -75,7 +75,7 @@ internal QCallModule(ref System.Reflection.Emit.ModuleBuilder module)
         internal QCallAssembly(ref System.Reflection.RuntimeAssembly assembly)
         {
             _ptr = Unsafe.AsPointer(ref assembly);
-            _assembly = assembly.GetUnderlyingNativeHandle();
+            _assembly = assembly?.GetUnderlyingNativeHandle() ?? IntPtr.Zero;
```

Comment:
Could you please update `QCallTypeHandle` constructor in this file to follow the same style?